tensorflow/inference/docker/2.2/py3/Dockerfile.cpu

```diff
@@ -38,11 +37,21 @@ RUN apt-get update \
     git \
     wget \
     vim \
+    libssl1.1 \
+    openssl \
     libbz2-dev \
     liblzma-dev \
     libffi-dev \
     build-essential \
     zlib1g-dev \
+    libreadline-gplv2-dev \
+    libncursesw5-dev \
+    libssl-dev \
+    libsqlite3-dev \
+    tk-dev \
+    libgdbm-dev \
+    libc6-dev \
+    libbz2-dev \
```

Comment:
Why are these packages needed?

Reply:
Such errors were seen in from `pip install` during TF2 build
```
WARNING: pip is configured with locations that require TLS/SSL, however the ssl module in Python is not available.
974 |  
975 | Could not fetch URL https://pypi.org/simple/pip/: There was a problem confirming the ssl certificate: HTTPSConnectionPool(host='pypi.org', port=443): Max retries exceeded with url: /simple/pip/ (Caused by SSLError("Can't connect to HTTPS URL because the SSL module is not available.")) - skipping
976
```

These were due to the missing necessary packages for python ssl